triagebot.toml

```diff
@@ -36,4 +36,5 @@ label = "ICEBreaker-Cleanup-Crew"
 
 [prioritize]
 label = "I-prioritize"
+prioritize_on = ["regression-from-stable-to-beta", "regression-from-stable-to-nightly"]
```

Comment:
Hm, probably stable-to-stable too?

Reply:
We're not currently prioritizing all of those right now, maybe we could start slowly starting to do so. cc @pnkfelix

Reply:
Actually, let's add those this will fire the request for prioritization for new ones. We can slowly update the old ones.

Reply:
Fixed.